kernel/src/main/java/org/kframework/kompile/DefinitionParsing.java

```diff
@@ -355,74 +346,150 @@ private Module resolveConfigBubbles(Definition def, Module inputModule, RuleGram
                 throw KEMException.compilerError("Module Map must be visible at the configuration declaration, in module " + module.name());
             }
             return Module(module.name(), (Set<Module>) module.imports().$bar(Set(mapModule)),
-                    (Set<Sentence>) module.localSentences().$bar(configDeclRules),
+                    (Set<Sentence>) module.localSentences().$bar(configDeclRules)
+                            .filter(s -> !(s instanceof Bubble && ((Bubble) s).sentenceType().equals(configuration))),
```

Comment:
We're returning a module here that only has configuration bubbles? What are we doing with this module? What about all the other sentences?

Reply:
The other way around. After config parsing, I'm removing all the bubbles that contain configurations.